slices/python3.10-venv.yaml

```
@@ -9,3 +12,9 @@ slices:
       - python3.10_standard
     contents:
       /usr/lib/python3.10/ensurepip/*.py:
+
+  copyright:
+    essential:
+      - python3.10_copyright
+    contents:
+      /usr/share/doc/python3.10-venv/: { symlink: /usr/share/doc/python3.10/ }
```

```suggestion
      /usr/share/doc/python3.10-venv/:
```

This already comes as a symlink from the package.

```sh
lrwxrwxrwx root/root         0 2024-11-07 02:22 ./usr/share/doc/python3.10-venv -> python3.10
```